packages/runtime-core/src/components/BaseTransition.ts

```diff
@@ -238,7 +238,8 @@ const BaseTransitionImpl: ComponentOptions = {
           // return placeholder node and queue update when leave finishes
           leavingHooks.afterLeave = () => {
             state.isLeaving = false
-            instance.update()
+            // #6835
+            instance.update.active !== false && instance.update()
```

Comment:
```suggestion
            if (instance.update.active) {
              instance.update()
            }
```

Reply:
Thanks for the suggestion, but this will cause the test to report an error because **instance.update.active** may be undefined and the component is required to be updated in this case.

Reference:
https://github.com/vuejs/core/blob/6aaf8efefffdb0d4b93f178b2bb36cd3c6bc31b8/packages/runtime-core/src/scheduler.ts#L233

Reply:
```suggestion
            if (instance.update.active !== false) {
              instance.update()
            }
```
Modified.